### ISSUE TITLE
[Spike] Investigate Eventsourcing event storage and retrieval in DBR 

### ISSUE THREAD

**Issue by kristianschneider** [Energinet-DataHub/geh-aggregations#360]
Prerequisite: We have a dataframe with the event. For example a meteringPointCreated event.

Store incoming events into delta lake.

Ensure partitioning strategy for each master data type.
 Meteringpoint
 Charge
 ChargeLink
 Prices
 MarketRoles

Create interface / function for pulling out events for each master data type with key. 

For example:
 Save_event(eventDF, ObjectId, ObjectType)
 Read_events(ObjectId,ObjectType) -> List[EventDF]

AC 1)  Workbook with POC
AC 2) Different approaches have been investigated and best concept has been agreed with team

**Comment by mknic** [comment]
A new issue takes over for this.
